internal_dev/TracingZError.ipf

```diff
@@ -1,4 +1,4 @@
-﻿#pragma TextEncoding = "UTF-8"
+#pragma TextEncoding = "UTF-8"
```

Comment:
Is this relevant for this PR?

Reply:
It is relevant to fix the current main branch to let this PR merge.